specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/stable/2025-03-01/examples/FleetMembers_Get.json

```diff
@@ -28,7 +28,13 @@
         "eTag": "kd30rkdfo49=",
         "properties": {
           "clusterResourceId": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/cluster-1",
-          "provisioningState": "Succeeded"
+          "provisioningState": "Succeeded",
+          "labels": [
+            {
+              "key": "environment",
+              "value": "production"
+            }
+          ]
```

Comment:
This looks neither like Kubernetes labels, nor like arm tags.
both Kubernetes Labels and Arm Tags are represented as a map, rather than an array of objects:

kubernetes labels:
```json
"metadata": {
  "labels": {
    "key1" : "value1",
    "key2" : "value2"
  }
}
```

arm tags
```json
    "location": "eastus2",
    "tags": {
        "test": "test"
    },
```